tools/perf/ucc_pt_coll_alltoall.cc

```diff
@@ -60,6 +64,11 @@ ucc_status_t ucc_pt_coll_alltoall::init_args(size_t single_rank_count,
                       free_dst, st);
         args.src.info.buffer = src_header->addr;
     }
+    if(UCC_IS_ONESIDED(args)){
+        args.src.info.buffer = comm->get_global_buffer(0);
```

Comment:
Maybe instead of 0, 1, or 2, define a name for them like `MEM_SEND_SEGMENT` ? Then can use like:
```
args.src.info.buffer = comm->get_global_buffer(MEM_SEND_SEGMENT);
```